modules/ui/fields/address.js

```diff
@@ -55,7 +55,14 @@ export function uiFieldAddress(field, context) {
                     dist = geoSphericalDistance(choice.loc, l);
                 }
 
-                const value = resultProp && d.tags[resultProp] ? d.tags[resultProp] : d.tags.name;
+                const value = resultProp && d.tags[resultProp]
+                    ? d.tags[resultProp]
+                    : d.tags['name:en']
+                        ? d.tags['name:en'] // Fallback to English name
```

Comment:
If I’m reading this code correctly, it doesn’t fall back to `name:en`, it prioritizes `name:en` over `name`. But not every region speaks English; that was just an example I gave. I gave some more details about how you might solve this problem in https://github.com/openstreetmap/iD/issues/10541#issuecomment-2504476250. However, the simplest solution may just be to truncate on the first semicolon and leave #10541 to address the linguistic problem separately.